### PR TITLE
add canary tests for tf serving container

### PR DESCRIPTION
**Pull request by jesterhazy**
*Description of changes:*

- add canary tests for tensorflow serving container

## Merge Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your pull request._

- [x] I have read the [CONTRIBUTING](https://github.com/aws/sagemaker-python-sdk/blob/master/CONTRIBUTING.md) doc
- [x] I have added tests that prove my fix is effective or that my feature works (if appropriate)
- [ ] I have updated the [changelog](https://github.com/aws/sagemaker-python-sdk/blob/master/CHANGELOG.rst) with a description of my changes (if appropriate)
- [x] I have updated any necessary [documentation](https://github.com/aws/sagemaker-python-sdk/blob/master/README.rst) (if appropriate)

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: tests/integ/__init__.py [hunk 1]
```diff
@@ -25,5 +25,10 @@
 PYTHON_VERSION = 'py' + str(sys.version_info.major)
 REGION = boto3.session.Session().region_name
 
+# https://code.amazon.com/packages/MaeveControlPlaneService/blobs/mainline/--/src/com/amazonaws/services/ironman/utils/InstanceTypesHelper.java
```
Comment (yangaws) [inline tests/integ/__init__.py]: I guess we should not link to internal resources on github.

File: tests/integ/test_tfs.py [hunk 2]
```diff
@@ -32,16 +33,19 @@ def tfs_predictor(instance_type, sagemaker_session, tf_full_version):
...
         yield predictor
 
 
-# @pytest.mark.continuous_testing
-# @pytest.mark.regional_testing
+@pytest.mark.continuous_testing
+@pytest.mark.regional_testing
+@pytest.mark.skipif(
+    tests.integ.REGION in tests.integ.HOSTING_P3_UNAVAILABLE_REGIONS,
+    reason='no ml.p3 instances in this region')
```
Comment (yangaws) [inline tests/integ/test_tfs.py]: I am not sure on this part. Will it skip c5 as well?

Comment (jesterhazy) [reply tests/integ/test_tfs.py]: oops yes. will fix

File: tests/integ/__init__.py (hunk 1 above)
Comment (jesterhazy) [reply tests/integ/__init__.py]: right

**Pull request merged**